### ISSUE TITLE
Blocking gets stuck if user no longer exists

### ISSUE THREAD

**Issue by nocha1n** [dmstern/likers-blocker#137]
If a user gets added to the list but cannot be found by the time it's time to block them (potentially through renaming their account, deleting their account, being suspended, etc.) then the addon will receive a 404 error but attempt to block them over and over and get stuck in a loop.

```json
{"errors":[{"code":50,"message":"User not found."}]}
```

The correct course of action would be to either skip the user or move it last in the list.

**Comment by dmstern** [comment]
Good point. Currently we just catch any error during the block request and requeue the user if the block did fail. Maybe we should rather additionally catch 404 errors and remove the certain user. What do you think?
